stable/doc_hardware.md

```diff
@@ -1 +1,107 @@
-coming soon...
\ No newline at end of file
+# Hardware documentation
+
+In this section we present a short documentation of our kit. More informations about the hardware can be found in our [website](http://echopen.org/doc-website/CAT-configuration/CFG-sweeping_probe/content.html) or download our [github folder](https://github.com/echopen/PRJ-medtec_kit), aquarium configuration (PRJ-medtec_kit/configuration/CFG-aquarium_kit/readme.md). We show synoptic and block diagrams of the kit and give some informations about daughter boards and modules.
+
+### Synoptic diagram
```

Comment:
discussed with @jerome2echopen : let's make a diagram focused on modules. The functional diagrams (general diagrams + pipeline of data processing) will be displayed in the high-level documentation